### ISSUE TITLE
1 hp PLAY minions + autocannon

### ISSUE THREAD

**Issue by Qudaci** [jeremylarner/keyteki#53]
when You have 'autocannon' on board, and you play a 1hp minion with a 'play' ability (eg. Urchin), the minion is immediately destroyed without activating its 'play' ablity (and it should. I asked around)

**Comment by BraccaDaxxa** [comment]
The same problem exists with Pingle Who Annoys as with Autocannon. The Play effect should happen for 1 HP creatures like Restriguntus and Skippy Timehod.

**Comment by seajaysec** [comment]
Yeah this is definitely violating the actual rules of the game. This alt ruling changes the meta a lot.

**Comment by seajaysec** [comment]
I saw #159 and #128 reported the same bug and were "fixed", but just played a game where this bug still exists.
